geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/sortedset/AbstractZUnionStoreIntegrationTest.java

```diff
@@ -234,20 +235,20 @@ public void shouldUnionize_givenBoundaryScoresAndWeights() {
 
     jedis.zunionstore(NEW_SET, new ZParams().weights(1), KEY1);
 
-    Set<Tuple> results = jedis.zrangeWithScores(NEW_SET, 0, scores.size());
-    assertThat(results).containsExactlyElementsOf(expectedResults);
+    List<Tuple> results = jedis.zrangeWithScores(NEW_SET, 0, scores.size());
+    assertThat(results).containsExactlyInAnyOrderElementsOf(expectedResults);
```

Comment:
Please keep all of these assertions as `containsExactlyElementsOf` if possible (I ran a quick test and looks like it's OK). Since `ZRANGESCORE` is supposed to return an ordered list, we should check that.

Reply:
These tests should also be using a List for their `expectedResults` rather than a set, since that's what Jedis returns now. This is another case where simple find/replace can make the conversion fairly painless.